run_analysis.py

```diff
@@ -1,109 +1,129 @@
-
-# choose the right rendering backend, depending on whether we're displaying
-# to the screen
-if not config.getboolean('plot','displayToScreen'):
-    mpl.use('Agg')
-import matplotlib.pyplot as plt
-
-# these only get imported after we have the right MPL renderer selected
-from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
-from mpas_analysis.ocean.sst_timeseries import sst_timeseries
-#from mpas_analysis.ocean.nino34_timeseries import nino34_timeseries
-#from mpas_analysis.ocean.mht_timeseries import mht_timeseries
-#from mpas_analysis.ocean.moc_timeseries import moc_timeseries
-from mpas_analysis.ocean.sst_modelvsobs import sst_modelvsobs
-
-from mpas_analysis.sea_ice.timeseries import seaice_timeseries
-from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
-
-
-#GENERATE OCEAN DIAGNOSTICS
-if config.getboolean('ohc_timeseries','generate'):
-    print ""
-    print "Plotting OHC time series..."
-    ohc_timeseries(config)
-
-if config.getboolean('sst_timeseries','generate'):
-    print ""
-    print "Plotting SST time series..."
-    sst_timeseries(config)
-
-if config.getboolean('nino34_timeseries','generate'):
-    print ""
-    print "Plotting Nino3.4 time series..."
-    #nino34_timeseries(config)
-
-if config.getboolean('mht_timeseries','generate'):
-    print ""
-    print "Plotting Meridional Heat Transport (MHT)..."
-    #mht_timeseries(config)
-
-
-if config.getboolean('moc_timeseries','generate'):
-    print ""
-    print "Plotting Meridional Overturning Circulation (MOC)..."
-    #moc_timeseries(config)
-    
-if config.getboolean('sst_modelvsobs','generate'):
-    print ""
-    print "Plotting 2-d maps of SST climatologies..."
-    sst_modelvsobs(config)
-
-
-#GENERATE SEA-ICE DIAGNOSTICS
-if config.getboolean('seaice_timeseries','generate'):
-    print ""
-    print "Plotting sea-ice area and volume time series..."
-    seaice_timeseries(config)
-
-if config.getboolean('seaice_modelvsobs','generate'):
-    print ""
-    print "Plotting 2-d maps of sea-ice concentration and thickness climatologies..."
-    seaice_modelvsobs(config)
-
-
-#GENERATE LAND-ICE DIAGNOSTICS
-
-
-if config.getboolean('plot','displayToScreen'):
-   plt.show()
+def path_existence(config, section, option, ignorestr=None): #{{{
+    inpath = config.get(section, option)
+    if not (os.path.isdir(inpath) or os.path.isfile(inpath)):
+        # assumes that path locations of ignorestr won't return an error, e.g.,
+        # ignorestr="none" is a key word to indicate the path or file is
+        # optional and is not needed
+        if inpath == ignorestr:
+            return False
+        errmsg = "Path %s not found. Exiting..." % inpath
+        raise SystemExit(errmsg)
+    return inpath #}}}
+
+def analysis(config): #{{{
+
+    # Checks on directory/files existence:
+    indir = path_existence(config, 'paths', 'archive_dir_ocn')
+
+    if config.get('case', 'ref_casename_v0') is not 'None':
```

Comment:
@milenaveneziani, does this mean that the config file recognized it as the special character `None`?  It sounds like this may be the case.  This isn't a big deal either way and I reverted it.  Thanks for catching this potential issue.